.cirrus.yml

```diff
@@ -0,0 +1,23 @@
+smoke_task:
+    env:
+       CIRRUS_CLONE_DEPTH: 10
+    setup_script:
+       - dnf -y install python3 git xz tcpdump nc iproute iputils gcc python3-devel qemu-kvm qemu-img
+       - python3 -m pip install $AVOCADO_SOURCE
```

Comment:
Shouldn't the `$AVOCADO_SOURCE` use quotations?

Reply:
It doesn't have to... works fine as is.

Reply:
at this point yes, but with the `<` I guess you'll have to face consequences (unless they escape it internally)